http-server/src/main/java/io/airlift/http/server/DropForwardedRequestCustomizer.java

```diff
@@ -0,0 +1,46 @@
+package io.airlift.http.server;
+
+import org.eclipse.jetty.http.HttpField;
+import org.eclipse.jetty.http.HttpFields;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.Request;
+
+import java.util.Set;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+
+public class DropForwardedRequestCustomizer
```

Comment:
This customizer removes X-Forwarded- headers from the request. I followed similar customizer to see how to mutate headers and wrap request without copying all the fields.